tests/bears/BearTest.py

```diff
@@ -463,6 +474,32 @@ def test_bear_with_specific_language(self):
         self.assertEqual(str(result), 'Hypertext Markup Language 5.1')
         self.check_message(LOG_LEVEL.DEBUG)
 
+    # Mock test added to solve the coverage problem by DebugBearsTest
+    @patch('pdb.Pdb.do_continue')
+    def test_custom_continue(self, do_continue):
+        arg = {}
+        Debugger().do_quit(arg)
+        pdb.Pdb.do_continue.assert_called_once_with(arg)
+        self.assertEqual(Debugger().do_quit(arg), 1)
+
+    @patch('coalib.bears.Bear.Debugger.runcall', side_effect=((1, 2), 1, 2))
+    def test_debug_run_with_return(self, runcall):
+        section = Section('name')
+        my_bear = TestOneBear(section, self.queue, debugger=True)
```

Comment:
do you actually need `TestOneBear`? Since you never invoke its `run` method, because it's stubbed away with `side_effect`. Maybe just use a plain `Bear` and write a comment that `side_effect` effectively "implements `run`".